src/components/GithubPopularRepositories.tsx

```diff
@@ -55,6 +56,20 @@ const GithubPopularRepositories: FC<Props> = ({ repositories }) => {
                   marginLeft="auto"
                 />
               </Flex>
+              <Flex marginTop={1}>
+                <Flex alignItems="center">
+                  <AiOutlineStar />
+                  <Text color="GrayText" paddingLeft={1} paddingRight={5}>
+                    {repository.stargazerCount}
+                  </Text>
+                </Flex>
+                <Flex alignItems="center">
+                  <BiGitRepoForked />
+                  <Text color="GrayText" paddingLeft={1}>
+                    {repository.forkCount}
+                  </Text>
+                </Flex>
+              </Flex>
```

Comment:
This should be below the description.

Reply:
ok

Reply:
@mayukh551 , Can you update the screenshots?

Reply:
@mayukh551, I meant in the PR description.

Reply:
Oh, ok

Reply:
@Sachin-chaurasiya updated the Screenshots